Fix Broken Links (#840)
plone/training: docs/effective-volto/getting-started/project.md

### nvm, Node.JS, Yeoman, and Yarn

> First install `nvm` and latest Node.JS according to the [Plone documentation](https://6.docs.plone.org/install/create-project.html#pre-requisites-for-installation).

Reviewer: ```suggestion
First install `nvm` and latest Node.JS according to the [Plone documentation](https://6.docs.plone.org/install/create-project.html#nvm).
```

> After that, install Yeoman according to the [Plone documentation](https://6.docs.plone.org/install/create-project.html#pre-requisites-for-installation).

Reviewer: ```suggestion
After that, install Yeoman according to the [Plone documentation](https://6.docs.plone.org/install/create-project.html#yeoman-and-the-volto-boilerplate-generator).
```

> Finally, install `yarn` according to the [Plone documentation](https://6.docs.plone.org/install/create-project.html#pre-requisites-for-installation).

Reviewer: ```suggestion
Finally, install `yarn` according to the [Plone documentation](https://6.docs.plone.org/install/create-project.html#yarn).
```
